app/uk/gov/hmrc/brm/controllers/BRMBaseController.scala

```diff
@@ -17,23 +17,23 @@
 package uk.gov.hmrc.brm.controllers
 
 import play.api.libs.json.JsValue
-import play.api.mvc.{Request, Result}
+import play.api.mvc.{ControllerComponents, Request, Result}
 import uk.gov.hmrc.brm.audit.{BRMDownstreamAPIAudit, MatchingAudit, TransactionAuditor}
 import uk.gov.hmrc.brm.models.brm.Payload
 import uk.gov.hmrc.brm.models.matching.MatchingResult
-import uk.gov.hmrc.brm.utils.CommonUtil._
-import uk.gov.hmrc.brm.utils.{BRMException, HeaderValidator}
-import uk.gov.hmrc.play.microservice.controller.BaseController
+import uk.gov.hmrc.brm.utils.{BRMException, CommonUtil, HeaderValidator}
 import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
 
-trait BRMBaseController extends BaseController with BRMException {
+abstract class BRMBaseController(cc: ControllerComponents) extends BackendController(cc) with BRMException {
```

Comment:
could this one not just be a trait with a val cc: ControllerComponents in the body? I'm genuinely curious, I didn't think putting this down as an abstract class is necessary

Reply:
Yes it could, and no it's not 👍